far/viewer.cpp

```diff
@@ -1007,20 +1011,25 @@ void Viewer::SetStatusMode(int Mode)
 	ShowStatusLine=Mode;
 }
 
-
-static bool is_word_div(const wchar_t ch)
+static bool is_word_div(const wchar_t ch, const string& word_div)
 {
-	static const wchar_t extra_div[]{ encoding::bom_char, encoding::replace_char };
-	return !ch || std::iswspace(ch) || contains(Global->Opt->strWordDiv.Get(), ch) || contains(extra_div, ch);
+	return !ch || std::iswspace(ch) || contains(word_div, ch);
 }
 
-static bool wrapped_char(const wchar_t ch)
+static string get_word_div()
 {
-	return IsBlankOrEos(ch) || contains(L",;>)"sv, ch);
+	static constexpr std::array extra_word_div{ encoding::bom_char, encoding::replace_char };
+
+	auto word_div{ Global->Opt->strWordDiv.Get() };
+	word_div.append(extra_word_div.cbegin(), extra_word_div.cend());
+	return word_div;
 }
 
 void Viewer::ReadString(ViewerString *pString, int MaxSize, bool update_cache)
 {
+	auto WordDiv{ Global->Opt->strWordDiv.Get() };
```

Comment:
const maybe?
Maybe even const& - we probably don't need a copy here?

Reply:
Oh! I somehow thought `Get()` returns `string`; `const&` it is!